charts/airbyte-server/templates/service.yaml

```diff
@@ -2,7 +2,7 @@
 apiVersion: v1
 kind: Service
 metadata:
-  name: {{ include "common.names.fullname" . }}
+  name: airbyte-server-svc
```

Comment:
Just wonder why we hard-coded the service names. This kinda messed up the branch deployments :|